seed-element.html

```diff
@@ -1,97 +1,110 @@
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
 -->
 <link rel="import" href="../polymer/polymer.html">
 
 <!--
-Element providing solution to no problem in particular.
+An element providing a solution to no problem in particular.
 
-##### Example
+Example:
 
     <seed-element></seed-element>
 
-@element seed-element
-@blurb Element providing solution to no problem in particular.
-@status alpha
-@homepage http://polymerlabs.github.io/seed-element
 -->
-<polymer-element name="seed-element" attributes="notitle author">
+<dom-module id="seed-element">
 
-  <template>
-
-    <link rel="stylesheet" href="seed-element.css">
+  <style>
+    seed-element {
+      display: block;
+      box-sizing: border-box;
+    }
+  </style>
 
+  <template>
     <h1>Hello from seed-element</h1>
     <content></content>
-
   </template>
 
-  <script>
-
-    Polymer({
-      /**
-       * The `author` attribute sets an initial author
-       *
-       * @attribute author
-       * @type string
-       * @default 'Dimitri Glazkov'
-       */
-      author: 'Dimitri Glazkov',
+</dom-module>
 
-      /**
-       * `fancy` is a property that does something fancy.
-       *
-       * @property fancy
-       * @type bool
-       * @default false
-       */
-      fancy: false,
+<script>
 
-      ready: function() {
-        // Ready is a lifecycle callback.
-        // You can do setup work in here.
-        // More info: http://www.polymer-project.org/docs/polymer/polymer.html#lifecyclemethods
-      },
+  Polymer({
 
-      /**
-       * The `sayHello` method will return a greeting.
-       *
-       * @method sayHello
-       * @param {String} greeting Pass in a specific greeting.
-       * @return {String} Returns a string greeting.
-       */
-      sayHello: function(greeting) {
-        var response = greeting || 'Hello World!';
-        return 'seed-element says, ' + response;
-      },
+    is: 'seed-element',
 
+    published: {
       /**
-       * The `seed-element-lasers-success` event is fired whenever we
-       * call fireLasers.
-       *
-       * @event seed-element-lasers-success
-       * @param {Object} detail
-       *   @param {string} detail.sound An amazing sound.
+       * The `author` attribute sets an initial author.
        */
+      author: String,
 
       /**
-       * The `fireLasers` method will fire the lasers. At least
-       * it will dispatch an event that claims lasers were fired :)
-       *
-       * @method fireLasers
+       * `fancy` indicates that the element should don a monocle and tophat,
+       * while checking its pocket watch.
        */
-      fireLasers: function() {
-        this.fire('seed-element-lasers-success', { sound: 'Pew pew pew!' });
-      }
-
-    });
-
-  </script>
-
-</polymer-element>
+      fancy: Boolean,
+    },
+
+    // Element Lifecycle
+
+    configure: function() {
+      // `configure` is the hook to set up any initial state for the element. It
```

Comment:
Explaining `configure`, `ready` etc. in inline comment form is extremely useful here. Thanks for doing this!